ruby/pom.xml

```diff
@@ -88,7 +88,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java-util</artifactId>
-            <version>3.13.0</version>
+            <version>3.18.0</version>
```

Comment:
I think you'll need to add an entry to this:
https://github.com/protocolbuffers/protobuf/blob/master/update_version.py

to ensure this stays updated.

Reply:
Done - thanks for the pointer.